docs/topics/multiplatform-mobile/multiplatform-mobile-setup.md

```diff
@@ -102,18 +104,19 @@ To make sure everything works as expected, install and run the KDoctor tool:
       </ul>
    </def>
    <def title="Kotlin plugins">
-        <list>
-            <ul>
-               <li>Make sure that both the Kotlin and Kotlin Mobile Multiplatform plugins are installed and enabled. In Android Studio, select <strong>Settings/Preferences | Tools | Plugins</strong> and then
-              verify that you have both plugins enabled in the <strong>Installed</strong> tab. If either is missing, search <strong>Marketplace</strong> and install it.
-              </li>
-              <li>
-                  <p>The current version of the Kotlin Multiplatform Mobile plugin may be incompatible with your version of Kotlin. Refer to the <a href="https://kotlinlang.org/docs/multiplatform-mobile-plugin-releases.html#release-details">compatibility table</a>.</p>
-                  <p>To update the Kotlin version, in Android Studio select <strong>Tools | Kotlin | Configure Kotlin Plugin Updates</strong>.</p>
-                  <p>To update the Kotlin Multiplatform Mobile plugin, select <strong>Settings/Preferences | Tools | Plugins</strong>. Next to the plugin name, click <strong>Update</strong>.</p>
-              </li>
-            </ul>
-         </list>
+         <chunk>
+            <p><strong>Kotlin Multiplatform Mobile plugin</strong></p>
+               <ul>
+                  <li>Make sure that the Kotlin Mobile Multiplatform plugin is installed and enabled. On the Android Studio welcome screen, select <menupath>Plugins | Installed</menupath>. Verify that you have the plugin enabled. If it's not in the <control>Installed</control> list, search <control>Marketplace</control> and install it.</li>
```

Comment:
Search for it in Marketplace?

Reply:
That what I had, but copyeditors suggested this wording